src/components/Link/Link.svelte

```diff
@@ -0,0 +1,9 @@
+<script>
+  export let href;
+  export let text;
+</script>
+
+<style type="text/postcss">
+</style>
+
+<a class="text-link hover:underline" href="{href}">{text}</a>
```

Comment:
I would @apply these classes in style-tag.

Reply:
Changed.